cli/commands/describetype/describe_type.go

```diff
@@ -0,0 +1,171 @@
+package describetype
+
+import (
+	"errors"
+	"fmt"
+	"io"
+	"strconv"
+
+	corev2 "github.com/sensu/sensu-go/api/core/v2"
+	"github.com/sensu/sensu-go/cli"
+	"github.com/sensu/sensu-go/cli/client/config"
+	"github.com/sensu/sensu-go/cli/commands/helpers"
+	"github.com/sensu/sensu-go/cli/elements/table"
+	"github.com/sensu/sensu-go/cli/resource"
+	"github.com/sensu/sensu-go/types"
+	"github.com/spf13/cobra"
+)
+
+var description = `sensuctl describe-type
+
+Describe a type by its fully qualified name:
+$ sensuctl describe-type core/v2.CheckConfig
+
+The command also supports describing types by their short names (for core/v2 types):
+$ sensuctl describe-type checks,handlers
+
+You can also use the 'all' qualifier to describe all available types:
+$ sensuctl describe-type all
+`
+
+type apiResource struct {
+	Name       string `json:"name"`
+	ShortName  string `json:"short_name"`
+	APIVersion string `json:"api_version"`
```

Comment:
This ends up being rendered as "apiversion" instead of "api_version" when output as YAML. Since the `sensuctl create` parser wants `api_version`, I feel it should be consistent here.